pkg/annotations/image.go

```diff
@@ -128,3 +126,30 @@ func CreateImageAnnotations(obj interface{}, name string, count int) map[string]
 
 	return newAnnotations
 }
+
+// ShortenLabelContent will ensure the data is less than the 63 character limit and doesn't contain
+// any characters that are not allowed
+func ShortenLabelContent(data string) string {
+	newData := RemoveRegistryInfo(data)
+
+	// Label values can not be longer than 63 characters
+	if len(newData) > 63 {
+		newData = newData[0:63]
+	}
+
+	return newData
+}
+
+// RemoveRegitryInfo will take a string and return a string that removes any registry name information
```

Comment:
I think travis is failing because of the typo in this comment.

Reply:
Yep. :) Fixed